visualisations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -0,0 +1,148 @@
+from webviz_plotly import Plotly
+import pandas as pd
+import matplotlib.cm as cm
+
+color_scheme = cm.get_cmap('Set1')
+
+
+def color_spread(n):
+    colorlist = []
+    for i in range(n):
+        single_color = color_scheme(float(i) / n)
+        formatted_color = []
+        for y in single_color[:-1]:
+            formatted_color.append(str(y))
+        colorlist.append(formatted_color)
+    return colorlist
+
+
+def format_color(color, opacity):
+    return "({},{})".format(','.join(color), opacity)
```

Comment:
This function should return "rgba(r,g,b,a)"  as this is the standard format for colors and every invocation of this function adds this prefix.